pennylane/templates/layer.py

```diff
@@ -31,9 +31,6 @@ def _preprocess(args, depth):
 
     for arg in args:
 
-        if qml.tape_mode_active():
-            arg = qml.math.angle(arg)
-
```

Comment:
@mariaschuld I have _no_ idea why this was here, is it a bug? I have deleted it as the `layer` tests would otherwise fail.

Reply:
Yo I have no idea. It is not even returned. This must have been an accidental copy and paste?

Reply:
:thinking: